forte/data/data_store.py

```diff
@@ -234,8 +235,8 @@ def _get_type_info(self, type_name: str) -> Dict[str, Any]:
             dynamic import is disabled.
         """
         # check if type is in dictionary
-        if type_name in self._type_attributes:
-            return self._type_attributes[type_name]
+        if type_name in DataStore._type_attributes:
```

Comment:
You may change the `self._type_attributes` in docstrings in line 153, 207, 209 and 928.